sdk/servicebus/service-bus/review/service-bus.api.md

```diff
@@ -75,31 +112,46 @@ export interface MessageHandlerOptions {
     maxMessageAutoRenewLockDurationInSeconds?: number;
 }
 
-export { MessagingError }
-
 // @public
-export interface OnError {
-    (error: MessagingError | Error): void;
+export interface MessageHandlers<ContextT> {
+    processError(err: Error): Promise<void>;
+    processMessage(message: ReceivedMessage, context: ContextT): Promise<void>;
 }
 
 // @public
-export interface OnMessage {
-    (message: ServiceBusMessage): Promise<void>;
-}
+export type MessageIterator<ContextT> = AsyncIterable<MessageAndContext<ContextT>>;
+
+export { MessagingError }
 
-// Warning: (ae-forgotten-export) The symbol "Client" needs to be exported by the entry point index.d.ts
-//
 // @public
-export class QueueClient implements Client {
+export interface NonSessionReceiver<LockModeT extends "peekLock" | "receiveAndDelete"> {
     close(): Promise<void>;
-    createReceiver(receiveMode: ReceiveMode): Receiver;
```

Comment:
should be `getReceiver`, `getSender` (not create)

Reply:
My vote is for `create` if these methods create new senders and receivers every time they are invoked

Reply:
To my recollection, @bterlson mentioned this naming aligns with storage?

Reply:
Yep - https://github.com/Azure/azure-sdk-for-net/blob/master/sdk/storage/Azure.Storage.Blobs/src/BlobServiceClient.cs#L349

Reply:
I think getSender/getReceiver sounds more streamlined. As a user, it is slightly worse to have to create a ServiceBusClient, and then have to create something else from that. Since we are sharing the underlying connection, I don't think it is too much of a stretch to say that we are "getting" something rather than creating.

Reply:
Now one thing that I believe someone brought up is whether you get the _same_ receiver if you call getReceiver() more than once (ie, does this thing create a new receiver each time or is it just `get`ing the existing receiver).

I'm throwing that in there as a point that favors 'create'.

Reply:
I think of it as more like, you are getting a receiver for the entity name you specify. I don't think this implies that it has to be the same instance.

Reply:
Went with 'get' to match what everyone else is doing.